packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -57,6 +59,22 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   };
 }
 
+export interface TabsContextProps {
+  variant: 'default' | 'light300';
+}
+
+const TabsContext = React.createContext<TabsContextProps>({
+  variant: 'default'
+});
+
+export const TabsContextProvider = TabsContext.Provider;
+export const TabsContextConsumer = TabsContext.Consumer;
+
+const variantStyle = {
+  default: '',
+  light300: styles.modifiers.colorSchemeLight_300
```

Comment:
I don't think we've ever postpended a number to a variant. If we're ever planning on using another "light" value I think this is good, though.

Reply:
I discussed it above, the issue with having just light is that its really misleading because this colorScheme is not white but light grey.

On the other hand, the PageSection component exposes also light variant, which is however really white, in contrast to this one.

For this reason I ended up being more fond of this solution with the number suffix.